chesster.js

```diff
@@ -1401,3 +1427,179 @@ function result_reply_missing_pairing(bot, message){
 function result_reply_updated(bot, message, result){
     bot.reply(message, "Got it. @" + result.white.name + " " + result.result + " @" + result.black.name);
 }
+
+
+
+/* game link parsing */
+
+//given a gamelink_id, use the lichess api to get the game details
+//pass the details to the callback as a JSON object
+function fetch_game_details(gamelink_id, callback){
+    const http = require('http');
+    var url = "http://en.lichess.org/api/game/" + gamelink_id;
```

Comment:
This is where I was thinking it was done again, but this is a different call. Again, you are correct. Carry on.